src/main/scala/uk/gov/hmrc/wco/dec/inventorylinking/consolidation/request/InventoryLinkingConsolidationRequest.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.wco.dec.inventorylinking.consolidation.request
+
+import java.io.StringWriter
+import java.util.Properties
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.dataformat.xml.annotation.{JacksonXmlProperty, JacksonXmlRootElement}
+import uk.gov.hmrc.wco.dec.MetaData
+import uk.gov.hmrc.wco.dec.inventorylinking.common.UcrBlock
+import uk.gov.hmrc.wco.dec.inventorylinking.movement.request.InventoryLinkingMovementRequest
+import uk.gov.hmrc.wco.dec.utilities.JacksonMapper
+
+import scala.collection.JavaConverters._
+
+object InventoryLinkingConsolidationRequest extends JacksonMapper {
+  final val inventoryLinking = "http://gov.uk/customs/inventoryLinking/v1"
+
+  def fromXml(xml: String): InventoryLinkingConsolidationRequest = _xml.readValue(xml, classOf[InventoryLinkingConsolidationRequest])
+
+  def fromProperties(props: Map[String, String]): MetaData = {
```

Comment:
whats the need for MetaData?
```suggestion
  def fromProperties(props: Map[String, String]): MetaData = {
```

Reply:
Missed it during copy-pasting. Thanks!